### DISCUSSION TITLE
Adding Browserstack device name in Allure Report

### DISCUSSION THREAD

**Discussion by sayantan-qa** [allure-framework/allure2#1298] [General Discussion]
I am running my test in Browserstack. I am using webdriverio and Allure as reporting tool. The report getting generated but Browserstack device name is not showing up in the Allure Report. Can anyone help?

**Comment by eroshenkoam** [comment]
You can provide this information via environment.properties file:
https://docs.qameta.io/allure/#_environment

You should put this file into allure-results directory and this information will be available on Overview page.

**Reply by BorisOsipov** [reply]
@sayantan-qa can you share screenshot and reporter version?
wdio-allure-reporter adds device name as test parameter automatically.

**Reply by sayantan-qa** [reply]
After changing to the latest version("@wdio/allure-reporter": "7.6.1",) , I am getting an error. The error I am getting is **:@wdio/local-runner: Failed launching test session: Error: ENOENT: no such file or directory, mkdir 'android-report-bs/allure-results'**

My package.json:
"devDependencies": {
    "@types/jasmine": "3.5.10",
    "@types/node": "13.13.4",
    "@wdio/allure-reporter": "7.6.1",
    "@wdio/appium-service": "^6.1.0",
    "@wdio/cli": "6.1.9",
    "@wdio/jasmine-framework": "6.1.8",
    "@wdio/local-runner": "6.1.9",
    "@wdio/browserstack-service": "6.1.10",
    "@wdio/selenium-standalone-service": "6.0.16",
    "@wdio/spec-reporter": "6.1.5",
    "@wdio/sync": "6.1.8",
    "axe-core": "3.3.2",
    "jasmine": "3.5.0",
    "jasmine-ts-console-reporter": "3.1.1",
    "prettier": "1.18.2",
    "rimraf": "3.0.2",
    "ts-node": "8.9.1",
    "tsconfig-paths": "3.9.0",
    "typescript": "3.8.3",
    "webdriverio": "6.1.9"
  },
  "engines": {
    "node": ">=7.6.0"
  },
  "dependencies": {}

I am not sure why this is coming. This folder should get created only after the test has run. This error was not there when I was using 6.1.5.

**Reply by BorisOsipov** [reply]
@sayantan-qa you can not update only reporter from v6 to v7. you have to update all wdio dependencies see https://webdriver.io/docs/v7-migration
